vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -42,8 +42,7 @@ function Get-ContainerImages
     $imagesToPull = @(
```

Comment:
$imagesToPull = @(
        "mcr.microsoft.com/windows/servercore:ltsc2019",
        "mcr.microsoft.com/windows/nanoserver:1809",
        "mcr.microsoft.com/oss/kubernetes/pause:1.4.0",
        "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.0.1-alpha.1-windows-1809-amd64",
        "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v1.2.1-alpha.1-windows-1809-amd64")

Reply:
Reference https://github.com/Azure/aks-engine/blob/master/vhd/packer/configure-windows-vhd.ps1

Reply:
Thanks for pointing this out.
This PR was based on v47.0.1,  do we need to also include all other changes in configure-windows-vhd.ps1 of master aks-engine here? and other pack files?

Reply:
Will sync to aks-engine master